packages/desktop-client/src/components/schedules/StatusBadge.js

```diff
@@ -9,45 +9,55 @@ import CalendarIcon from 'loot-design/src/svg/v2/Calendar';
 import ValidationCheck from 'loot-design/src/svg/v2/ValidationCheck';
 import FavoriteStar from 'loot-design/src/svg/v2/FavoriteStar';
 import CheckCircle1 from 'loot-design/src/svg/v2/CheckCircle1';
+import { useTranslation } from 'react-i18next';
 
 export function getStatusProps(status) {
-  let color, backgroundColor, Icon;
+  let color, backgroundColor, Icon, Title;
```

Comment:
`Title` isn’t being used as a component, so it doesn’t need to be uppercased.

```suggestion
  let title, color, backgroundColor, Icon;
```